Hogajama/hogajama-rs/src/test/java/com/gepardec/hogarama/rest/util/TestCaseDateUtil.java

```diff
@@ -61,7 +58,7 @@ public void testDateUtilTimeOK() {
 	public void testDateUtilTimeNOKHour() {
 		try {
 			DateUtil.getTime(99, 0, 0);
-			assertTrue("IllegalArgumentException was not thrown.", false);
+			assertTrue(false, "IllegalArgumentException was not thrown.");

```

Comment:
Maybe we should use assertThrows for this test and the following. This would increase readability.